pkgs/by-name/li/libmediainfo/package.nix

```
@@ -10,7 +14,7 @@ stdenv.mkDerivation rec {
   };
 
   nativeBuildInputs = [ autoreconfHook pkg-config ];
-  buildInputs = [ zlib ];
+  buildInputs = [ zlib curl ];
```

```suggestion
  buildInputs = [
      zlib
  ] ++ lib.optionals curlSupport [
      curl
  ];
```
Maybe it would be better to use `lib.optionals`